health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -348,6 +390,222 @@ function UploadDataMapping({ formData, onSelect, currentCategories }) {
+    const phoneNumberKey = t(Schemas?.find((i) => i.description === "Phone Number")?.name);
+    if (data[phoneNumberKey] !== undefined) {
+        data[phoneNumberKey] = Number(data[phoneNumberKey]);
+    }
+
+    // Capacity conversion
+    const capacityKey = t(Schemas?.find((i) => i.description === "Capacity")?.name);
+    if (data[capacityKey] !== undefined) {
+        data[capacityKey] = Number(data[capacityKey]);
+    }
+    const ajv = new Ajv({ strict: false }); // Initialize Ajv
+    let validate = ajv.compile(translatedSchema[type]);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'validate' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)